Bugsnag/KSCrash/Source/KSCrash/Recording/Sentry/BSG_KSCrashSentry_NSException.m

```diff
@@ -130,9 +132,53 @@ void bsg_recordException(NSException *exception) {
 
         BSG_KSLOG_DEBUG(@"Calling main crash handler.");
         bsg_g_context->onCrash(crashContext());
+        
+        bsg_kscrashsentry_resumeThreads();
     }
 }
 
+// ============================================================================
+#pragma mark - iOS apps on macOS -
+// ============================================================================
+
+// iOS apps behave a little differently when running on macOS via Catalyst or
+// on Apple Silicon. Uncaught NSExceptions raised while handling UI events get
+// caught by AppKit and are not propagated to NSUncaughtExceptionHandler or
+// std::terminate_handler, therefore we need another way to detect them...
```

Comment:
I'd add the feedback number here as a reference for future travelers, both to note that its related to an ongoing issue and to make it easier to know when to remove this code if the issue is ever addressed. (maybe also in the changelog?)